packages/@aws-cdk/aws-lambda/test/integ.bundling.ts

```diff
@@ -0,0 +1,43 @@
+import { App, CfnOutput, Construct, FileSystem, Stack, StackProps } from '@aws-cdk/core';
+import * as path from 'path';
+import * as lambda from '../lib';
+
+/**
+ * Stack verification steps:
+ * * aws cloudformation describe-stacks --stack-name cdk-integ-lambda-docker --query Stacks[0].Outputs[0].OutputValue
+ * * aws lambda invoke --function-name <output from above> response.json
+ * * cat response.json
+ * The last command should show '200'
+ */
+class TestStack extends Stack {
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    const assetPath = path.join(__dirname, 'python-lambda-handler');
+    const fn = new lambda.Function(this, 'Function', {
+      code: lambda.Code.fromAsset(assetPath, {
+        bundling: {
+          image: lambda.Runtime.PYTHON_3_6.bundlingDockerImage,
+          command: [
+            'bash', '-c', `
+            pip install -r requirements.txt -t /bundle &&
+            rsync -r . /bundle
+            `,
+          ],
```

Comment:
I would flip the order not to contaminate the source:

```suggestion
          command: [
            'bash', '-c', [
              'rsync -r . /bundle',
              'cd /bundle',
              'pip install -r requirements.txt -t .'
            ].join(' && ')
          ],
```

Reply:
The source is actually not "contaminated" because of the `-t /output´. Flipping will make it more obvious.